src/colorize.cr

```diff
@@ -128,15 +128,18 @@ module Colorize
   # ```
   #
   # NOTE: This is by default disabled on non-TTY devices because they likely don't support ANSI escape codes.
-  # This will also be disabled if the environment variable `TERM` is "dumb".
-  class_property? enabled : Bool = true
+  # This will also be disabled if the environment variable `TERM` is "dumb" or `NO_COLOR` contains any value.
```

Comment:
👀 The existing documentation is actually wrong. But that's a separate bug fix.

Reply:
Always mention what's wrong 😅 I guess you mean that the default doesn't respect `TERM` etc until an explicit call to `on_tty_only!`? 

It would probably be easier to reason about if we leave this as is or move the entire `on_tty_only!` logic in here.

Reply:
Yeah, sorry.
If you just include colorize and do nothing, it's enabled. The `TERM` variable only matters when `on_tty_only!` is called.
(This is not very user-friendly and should be changed, but it's what we have currently.)